김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/IO/RetrofitClass.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.mvvmstudy.IO
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+object RetrofitClass {
+    var retrofitService : IGithubService
+
+    init {
+        val interceptor = HttpLoggingInterceptor()
+        interceptor.level = HttpLoggingInterceptor.Level.BODY
+        val logger = OkHttpClient.Builder().addInterceptor(interceptor)
```

Comment:
스코프 함수로 줄일 수 있을 것 같아요! 변수명도 okHttpClient로 변경하는 것은 어떤가요!
```suggestion
          val okHttpClient = OkHttpClient.Builder().addInterceptor(
                 HttpLoggingInterceptor().apply { level = HttpLoggingInterceptor.Level.BODY }
        )
```

Reply:
줄이는게 좋을거 같네요 ㅎㅎ